.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,64 @@
+# Contributing to amrita_pyq
+
+Thank you for considering contributing to the amrita_pyq project! We welcome contributions in various forms, including bug reports, feature requests, and code contributions.
+
+## How to Contribute
+
+### Reporting Issues
+
+If you encounter any issues or have suggestions for improvements, please [open an issue](https://github.com/CSE-25/amrita_pyq/issues).
+
+### Forking the Repository
+
+1. Fork the repository by clicking the "Fork" button at the top right corner of the repository page.
+2. Clone the forked repository to your local machine:
+    ```bash
+    git clone https://github.com/YOUR_USERNAME/amrita_pyq.git
+    cd amrita_pyq
+    ```
+
+### Branching and Making Changes
+
+1. Create a new branch for your changes:
+    ```bash
+    git checkout -b <branch_name>
+    ```
+2. Make your changes in the new branch.
+3. Commit your changes with a meaningful commit message:
+    ```bash
+    git add .
+    git commit -m "Description of the changes"
+    ```
+4. Push your changes to your forked repository:
+    ```bash
+    git push origin <branch_name>
+    ```
+    
+### Submitting a Pull Request
+
+1. Once you push the changes to your repo, the `Contribute` Button will appear in Github.
+2. Click that button and you will be taken to the PR page. Describe what changes you have made and submit the PR.
+    <ol type="a">  
+        <li>Mention the issue link in the PR description.</li>  
+        <li>If the PR is still a work in progress, start the title with "[WIP]".</li>  
+        <li>If the PR is complete, start the title with "[COMPLETED]".</li>  
+        <li>Add "@Abhinav-ark" or "@Ashrockzzz2003" as reviewers.</li>  
+    </ol> 
+3. Wait for the PR to be reviewed by the maintainers.
+4. Make the required changes if the maintainers ask for them.
+5. Once the maintainers approve the PR, the PR will be merged.
+
+### Note
+
+If you make the contributions in October, you will be eligible for the [Hacktoberfest](https://hacktoberfest.com) event. Make sure you're registered on the [Hacktoberfest](https://hacktoberfest.com) website to be eligible for the badges from [Hacktoberfest](https://hacktoberfest.com) by contributing to this repo.
+
+
+### Code of Conduct
```

Comment:
Comment out the Code of Conduct part as it's not up yet. Don't remove it, just comment it out.